acquia/acquia-cms-starterkit
# ACMS-1677: Moving dependencies from Contrib to respective Starter Kit.

### Motivation
Fixes: [ACMS-1677](https://backlog.acquia.com/browse/ACMS-1677)

### Proposed changes
Moving dependencies from Contrib to respective Starter Kit.

### Alternatives considered
NA

### Testing steps
Install site with ACMS-1677 branch
Verify installation with all starterkits.
Verify site working fine.
